### PR TITLE
refactoring: Extract isGlimmerComponent to an utils file

### PR DESCRIPTION
**Pull request by dcyriller**
#### Description

This is a refactoring PR for glimmer printer. I want to use this util in a coming PR. But on the way, I'll have quite a few other functions to extract this utils file.

My plan is to submit one PR for this small change, another one to extract some other utils and last one using `isGlimmerComponent` util.

I don't know if you prefer to review small chunks one by one or if you prefer only one PR extracting all the things. Feel free to tell me.

#### Checklist

- [ ] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [ ] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [ ] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-handlebars/utils.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+function isGlimmerComponent(node) {
+  if (node.type !== "ElementNode") {
+    return false;
+  }
+
+  const tagFirstChar = node.tag && node.tag[0];
+  const isLocal = node.tag.indexOf(".") !== -1;
```
Comment (fisker) [inline src/language-handlebars/utils.js]: This will be a eslint error, when merge `master` into `next`,  how about commit to `next`

Comment (dcyriller) [reply src/language-handlebars/utils.js]: Out of curiosity: why https://github.com/prettier/prettier/pull/6967 did not target master?

Comment (fisker) [reply src/language-handlebars/utils.js]: https://github.com/prettier/prettier/pull/6968#issuecomment-554629174

File: src/language-handlebars/utils.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+function isGlimmerComponent(node) {
+  if (node.type !== "ElementNode") {
+    return false;
+  }
+
+  const tagFirstChar = node.tag && node.tag[0];
```
Comment (fisker) [inline src/language-handlebars/utils.js]: This part `node.tag && ` seems useless, you use `node.tag` bellow

**Pull request merged**